src/drivers/pwm_out/PWMOut.cpp

```diff
@@ -529,7 +547,7 @@ void PWMOut::Run()
 		ScheduleClear();
 		_mixing_output.unregister();
 
-		exit_and_cleanup();
+		//exit_and_cleanup();
```

Comment:
Delete or add a comment why it is commented out. Otherwise it's not clear.

Reply:
Done.